src/components/Input/index.js

```diff
@@ -72,13 +72,14 @@ const TextareaContainer = styled.div`
   }
 `
 
-const CustomInput = ({ textarea, name, label, placeholder, handleChange }) =>
-  !textarea ? (
+const CustomInput = ({ textarea, name, label, placeholder, handleChange }) => {
+  const placeholderText = placeholder || label
```

Comment:
That's not what I meant, I was saying that the value of the placeholder is the exact one which I write in the placeholder. 

If I want to add three dots ... then I can add them myself the component shouldn't add them automatically